code/modules/admin/verbs/adminjump.dm

```diff
@@ -143,25 +143,10 @@
 		return
 	var/area/A = input(usr, "Pick an area.", "Pick an area") in sortedAreas|null
 	if(A && istype(A))
-		feedback_add_details("admin_verb","SMOB") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
-		if(admin_forcemove(M, safepick(get_area_turfs(A))))
+		if(M.forceMove(safepick(get_area_turfs(A))))
+
 			log_admin("[key_name(usr)] teleported [key_name(M)] to [A]")
 			message_admins("[key_name_admin(usr)] teleported [key_name_admin(M)] to [A]")
 		else
 			src << "Failed to move mob to a valid location."
-
-/proc/admin_forcemove(mob/mover, atom/newloc)
-	if(!newloc)
-		return 0
-	if(mover.buckled)
-		mover.buckled.unbuckle_mob()
-	if(mover.buckled_mob)
-		mover.unbuckle_mob(force=1)
-	mover.loc = newloc
-	mover.on_forcemove(newloc)
-	return 1
-
-/mob/proc/on_forcemove(atom/newloc)
-	return

```

Comment:
Removed admin_forcemove(). It was pretty much a copypasta of forceMove(). The admin procs now use forceMove() instead.